pytorch_lightning/trainer/lr_finder.py

```diff
@@ -98,6 +98,7 @@ def lr_find(
             model: LightningModule,
             train_dataloader: Optional[DataLoader] = None,
             val_dataloaders: Optional[Union[DataLoader, List[DataLoader]]] = None,
+            datamodule: Optional[LightningDataModule] = None,
```

Comment:
lets move it at the end so we do not break compatibility...

Reply:
Done, thanks for pointing it out.